### PR TITLE
Offline Surface Tracker Data Format

### PR DESCRIPTION
**Pull request by marc-tonsen**
Added a description of the data format used by the offline surface tracker. Please review carefully and check correctness.

### CODE REVIEW

File: user-docs/pupil-player.md [hunk 1]
```diff
@@ -198,17 +198,45 @@ These plugins are simple unique plugins, that operate on the gaze data for analy
...
+  
+  + `heatmap_<surface_name>_<surface_id>.png` - Heatmap of gaze positions on the surface aggregated over the entire export.
+  + `fixations_on_surface_<surface_name>_<surface_id>.csv` - A list of fixations that have occured on the surface. 
+  + `gaze_positions_on_surface_<surface_name>_<surface_id>.csv` - A list of gaze datums on the surface. 
+  The values include the gaze point in two different coordinates systems.
+  `x_norm` and `y_norm` are coordinates between 0 and 1, where `(0,0)` is the bottom left corner of the surface and `(1,1)` is the top right corner.
+  `x_scaled` and `y_scaled` contain the same coordinates but scaled with the size defined for the surface.
+  + `srf_positons_<surface_name>_<surface_id>` - List of surface positions in 3D. 
+  The position is given as the 3D pose of the surface in relation to the current position of the scene camera.
+  `m_to_screen` is a matrix transforming coordinates from the camera coordinate system to the surface coordinate system.
```
Comment (marc-tonsen) [inline user-docs/pupil-player.md]: I would like to change the naming here to `m_to_surface` and `m_from_surface`. The word screen is not correct in many use cases.

File: user-docs/pupil-player.md [hunk 2]
```diff
@@ -198,17 +198,45 @@ These plugins are simple unique plugins, that operate on the gaze data for analy
...
+  + Recalculate gaze distributions - click the `(Re)calculate gaze distributions` button after specifying surface sizes. 
+  You should now see heatmaps in the Player window (if gaze positions were within your defined surfaces).
   + Export gaze and surface data - click `e` and all surface metrics reports will be exported and saved for your trim section within your `export` folder.
 
+All files generated by the `Offline Surface Detector` will be located in the subfolder `surfaces`. 
+The different reported metrics are:
+
+  + `surface_visibility.csv` - Overview of how many world camera frames each surface was contained in.
+  + `surface_gaze_distribution.csv` - Overview of how many gaze samples have been collected on each individual surface and outside of surfaces.
+  + `surface_events.csv` - List of gaze-enter and gaze-exit events for all surfaces.
```
Comment (mkassner) [inline user-docs/pupil-player.md]: @marc-tonsen  this is actually not gaze ent and gaze exit but surface enter and exit events.

File: user-docs/pupil-player.md (hunk 1 above)
Comment (mkassner) [reply user-docs/pupil-player.md]: fine with me.

**Comment by mkassner** [comment]
I have prematurely merged this @marc-tonsen  please make changes and a new PR for those changes.

**Pull request merged**